client/src/components/SearchBar/SearchBar.css

```diff
@@ -18,5 +18,5 @@
 
 .searchbox {
   width: 600px;
```

Comment:
Please use a responsive dimension for search box.

Reply:
the searchbox will be responsive once the window becomes smaller, check App.css for more details!